daemon/graphdriver/zfs/zfs.go

```diff
@@ -390,7 +390,7 @@ func (d *Driver) Get(id, mountLabel string) (_ containerfs.ContainerFS, retErr e
 	}
 
 	if err := mount.Mount(filesystem, mountpoint, "zfs", options); err != nil {
-		return nil, fmt.Errorf("error creating zfs mount of %s to %s: %v", filesystem, mountpoint, err)
+		return nil, err
```

Comment:
I think we should keep zfs in there (same with other graphdriver's error messages as well).

"error creating zfs mount" is much easier to search and find in code or to find other people reporting a similar error.

Reply:
fixed

Reply:
In general, though, I don't think having unique error strings to aid developers in bug report analysis is a good strategy. It would be much better to provide some context in error messages (like file name and line number where the error happens). This is the way I was doing it back in my C days (using preprocessor macros). Alas, logrus doesn't do it. Perhaps it will get this functionality one day? Could be possible with `debug.Stack()` or smth.

Reply:
"errors.WithStack" from "github.com/pkg/errors"... but still a good error message that signals where the error came from is important.